asv_bench/benchmarks/io/pickle.py

```diff
@@ -1,27 +0,0 @@
-import numpy as np
-from pandas import DataFrame, date_range, read_pickle
-import pandas.util.testing as tm
-
-from ..pandas_vb_common import BaseIO
-
-
-class Pickle(BaseIO):
-
-    def setup(self):
-        self.fname = '__test__.pkl'
-        N = 100000
-        C = 5
-        self.df = DataFrame(np.random.randn(N, C),
-                            columns=['float{}'.format(i) for i in range(C)],
-                            index=date_range('20000101', periods=N, freq='H'))
-        self.df['object'] = tm.makeStringIndex(N)
-        self.df.to_pickle(self.fname)
-
-    def time_read_pickle(self):
-        read_pickle(self.fname)
```

Comment:
This is the only part you would need to get rid of, I don't think you have to remove the whole Pickle Class?